src/openmp-validation-1-fixes.patch

```diff
@@ -42,5 +49,56 @@ diff -u OpenMP3.1_Validation/omp_my_sleep.h OpenMP3.1_Validation.new/omp_my_slee
  #include <sys/time.h>
  #include <time.h>
  #include <errno.h>
-Only in OpenMP3.1_Validation.new: ompts.log
-Only in OpenMP3.1_Validation.new: results.txt
+
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Tony Theodore <tonyt@logyst.com>
+Date: Sun, 23 Feb 2014 05:22:26 +1100
+Subject: [PATCH] add Batchfile with test to run on host
+
+
+diff --git a/all-tests-openmp-validation.bat b/all-tests-openmp-validation.bat
```

Comment:
`.gitattributes` was added in commit https://github.com/mxe/mxe/commit/b2bef6b006c41ca621470fbe3aa02b6f00848df1 but it does not provide the background.

IMHO, we can stay with consolidated patches, remove `.gitattributes` and revisit the problem if it happens again.